src/frontend/next/src/pages/LogoIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+
```

Comment:
@chrispinkney : The path should be `/src/components/LogoIcon/LogoIcon.tsx`. Also can you squash into 1 commit instead of 2?

Reply:
Done like dinner.